contracts/HangmanFactory.sol

```diff
@@ -46,7 +46,10 @@ contract HangmanFactory is ChainlinkClient, Ownable {
      */
     function requestCreateGame(bytes32 job_id, uint256 payment) public {
         //transfer LINK to this contract so it can request
-        require(linkERC20(linkToken).transferFrom(msg.sender, this, payment));
+        require(linkERC20(linkToken).transferFrom(msg.sender, this, payment), "Cannot tranfer link");
```

Comment:
I don't like this error message, but don't have anything better atm. Let's think of a better message.
This line is supposed to temporarily give this contract link before it provides it to the oracle.

Reply:
let's require() a check of balance and require that there is a transfer